README.md

```diff
@@ -2100,6 +2100,7 @@ Contains information about this system's video streaming capabilities.
 |`frequencyFraction`|Integer|False|The fractional part of the frequency for 101.7 is 7|
 |`band`|RadioBand|False||
 |`rdsData`|RdsData|False||
+|`hdRadioEnable`|Boolean|False|True if the hd radio is on, false is the radio is off|
```

Comment:
Should this say `false if the radio is off` (instead of `false is the radio is off`)?